### PR TITLE
Added firmware for 362001065279+U-WB05RT13V1.21.bin

### PR DESCRIPTION
**Pull request by KiralyCraft**
It seems my new AC won't bind using the default encryption key for scanning. However, the responses it gives from a plain `{"t": "scan"}` given to it's UDP port 7000, returns some data which can be decoded using the standard key. In there, my unit reports:

```
{"t": "pack", "i": 1, "uid": 0, "cid": "", "tcid": "", "pack": {"t": "dev", "cid": "9424b8b6b00f", "bc": "00000000000000000000000000000000", "brand": "gree", "catalog": "gree", "mac": "9424b8b6b00f", "mid": "10001", "model": "gree", "name": "", "lock": 0, "series": "gree", "vender": "1", "ver": "V2.0.0", "ModelType": "32772", "hid": "362001065279+U-WB05RT13V1.21.bin"}}
```

I got the firmware code from there, then used your CURL command. The model of my WiFi card is `GRJWB04-J`

### CODE REVIEW

File: 362001065279/U-WB05RT13V1.21-4307516678339185323.md
Comment (maxim-smirnov) [inline 362001065279/U-WB05RT13V1.21-4307516678339185323.md:1]: Hi there! Thank you for your contribution! Can you please add the link itself, like here https://github.com/maxim-smirnov/gree-wifimodule-firmware/blob/main/362001000835/U-CS532AX(MTK)V3-5351076961300064561.md
Just to make it clear, with the firmware code. Thank you!

**Comment by wangeris** [comment]
hello, may I ask what software did you use to make the query/call? Thank you in advance

**Comment by KiralyCraft** [comment]
> hello, may I ask what software did you use to make the query/call? Thank you in advance

First, I used this: https://github.com/cmroche/greeclimate with the `gree.py` script, calling it like `gree.py --discovery` 
You can also talk to your device directly using `netcat`, like `echo {"t": "scan"} | nc -u 192.168.0.131 7000` (my device was at `192.168.0.131`)

**Review by maxim-smirnov — APPROVED** [review]

File: 362001065279/U-WB05RT13V1.21-4307516678339185323.md
Comment (KiralyCraft) [reply 362001065279/U-WB05RT13V1.21-4307516678339185323.md:1]: Oh I didn't notice the link, thank you for adding it; I didn't get notifications for this for some reason

Comment (maxim-smirnov) [reply 362001065279/U-WB05RT13V1.21-4307516678339185323.md:1]: No problem :)

**Comment by wangeris** [comment]
ran netcat and decoded the response, got this: 

`{'t': 'dev', 'cid': '', 'bc': '00000000000000000000000000000000', 'brand': 'gree', 'catalog': 'gree', 'mac': '502cc6d11922', 'mid': '10001', 'model': 'gree', 'name': '', 'lock': 0, 
'series': 'gree', 'vender': '1', 'ver': 'V3.0.0'}`

seems like I'm unlucky and my unit does not have the firmware code encoded in the string

**Comment by KiralyCraft** [comment]
> seems like I'm unlucky and my unit does not have the firmware code encoded in the string

My unit's WiFi module is `WB05RT13V1.21`, and that is reflected in the name of the binary. Do you happen to have yours? Mine's written on a sticker on the module itself. Maybe your firmware is already dumped?

**Comment by wangeris** [comment]
> > seems like I'm unlucky and my unit does not have the firmware code encoded in the string
> 
> My unit's WiFi module is `WB05RT13V1.21`, and that is reflected in the name of the binary. Do you happen to have yours? Mine's written on a sticker on the module itself. Maybe your firmware is already dumped?

hey, my module is WB05RT13V1.09, just got an update to 1.23 via EU servers. Will try to fetch.

netcat scan decoded response now shows the firmware code and all

`{'t': 'dev', 'cid': '502cc6d11922', 'bc': '00000000000000000000000000000000', 'brand': 'gree', 'catalog': 'gree', 'mac': '502cc6d11922', 'mid': '10001', 'model': 'gree', 'name': '', 'lock': 0, 'series': 'gree', 'vender': '1', 'ver': 'V2.0.0', 'ModelType': '32792', 'hid': '362001065279+U-WB05RT13V1.23.bin'}`

**Pull request merged**